filament/src/ShadowMap.cpp

```diff
@@ -136,13 +136,15 @@ void ShadowMap::prepare(DriverApi& driver, SamplerGroup& sb) noexcept {
             TargetBufferFlags::DEPTH, dim, dim, 1,
             {}, { mShadowMapHandle }, {});
 
-    sb.setSampler(PerViewSib::SHADOW_MAP, {
-        mShadowMapHandle, {
-                    .filterMag = SamplerMagFilter::LINEAR,
-                    .filterMin = SamplerMinFilter::LINEAR,
-                    .compareFunc = SamplerCompareFunc::LE,
-                    .compareMode = SamplerCompareMode::COMPARE_TO_TEXTURE
-            }});
+    SamplerGroup::SamplerParams params;
```

Comment:
This should be fixed by #1706 -- you should be able to revert this whole block.

Reply:
I was able to use the original form except that the order of members has to be followed.